nixos/lib/eval-config.nix

```
@@ -90,7 +90,7 @@ let
       You have set specialArgs.pkgs, which means that options like nixpkgs.config
       and nixpkgs.overlays will be ignored. If you wish to reuse an already created
       pkgs, which you know is configured correctly for this NixOS configuration,
-      please import the `nixosModules.pkgsReadOnly` module from the nixpkgs flake or
+      please import the `nixosModules.readOnlyPkg` module from the nixpkgs flake or
```

```suggestion
      please import the `nixosModules.readOnlyPkgs` module from the nixpkgs flake or
```

I, uh, think it might still be typo'd 😅

---

Dang it! My dastardly plan to inflate my PR count was thwarted! /s